stdlib/public/Concurrency/CheckedContinuation.swift

```diff
@@ -251,6 +270,17 @@ public func withCheckedContinuation<T>(
   }
 }
 
+/// Calls the given throwing closure with a checked continuation.
+///
+/// - Parameters:
+///   - function: A string identifying the declaration that is the notional
+///     source for the continuation, used to identify the continuation in
+///     runtime diagnostics related to misuse of this continuation.
+///   - body: A closure that takes an `UnsafeContinuation` parameter.
+///     The closure must resume the continuation exactly once.
+///
+/// If the closure throws an error,
+/// this function rethrows that error.
```

Comment:
The distinction between `with*Continuation` and `with*ThrowingContinuation` is not whether the *closure* throws an error, but whether the *continuation* throws an error. Note that in neither form can the `body` throw an error directly. The continuation passed into the `Throwing` variants can resume the async task by invoking the `resume(throwing:)` method on it.

I think the summary text for all of these `with*Continuation` top-level functions should also note that they suspend the async task in progress, and that the continuation passed into the closure is not just "a" continuation but the continuation that resumes the current task. How about:

> Suspends the current async task, then calls the given closure with the task's continuation.

Reply:
Thanks — fixed in 55e8ec1fdc8.